src/javascript/utils.js

```diff
@@ -26,22 +26,26 @@ function checkObject (object) {
     (object === undefined)
 }
 
+function isInstanceOfType (instance, type) {
+  return Object.prototype.toString.call(instance) === `[object ${type}]`
+}
+
 function checkUniform (program, location) {
   return location instanceof WebGLUniformLocation &&
     location._program === program &&
     location._linkCount === program._linkCount
 }
 
 function isTypedArray (data) {
-  return data instanceof Uint8Array ||
-    data instanceof Uint8ClampedArray ||
-    data instanceof Int8Array ||
-    data instanceof Uint16Array ||
-    data instanceof Int16Array ||
-    data instanceof Uint32Array ||
-    data instanceof Int32Array ||
-    data instanceof Float32Array ||
-    data instanceof Float64Array
+  return isInstanceOfType(data, 'Uint8Array') ||
```

Comment:
Could you briefly describe the issues in node with checking if it's a typed array with `instanceof` ?

This looks fine, I'm more curious for myself – and perhaps there's a another, simpler yet still foolproof version.

Reply:
I added a comment to "isInstanceOfType". The issue I think was coming from Webpack, which was supplying some kind of polyfill that broke instanceof. I didn't 100% root cause the issue to the point where I know who was responsible, but I am certain that instanceof was failing in this case and comparing the type string did work properly for our case.